### ISSUE TITLE
Including Support of Multiple Trips Reminder

### ISSUE THREAD

**Issue by ankita-kalra** [systers/malaria-app-android#66]
Managing multiple trips effectively. There should be alarm IDs specific of the Trip ID. So multiple alarms are managed effectively.

**Comment by Shreya001** [comment]
Could you please explain the issue i would like to work on it please. @anubhakushwaha

**Comment by anubhakushwaha** [comment]
@Shreya001 Currently the app supports single trip, we can make multiple trips , I was suggesting making a class for a trip that would have all the details about the trip and as soon as a user makes a new trip we will simply have an object i.e at the end we will have a list of objects which need to be sorted by the dates, did you get it? And incase you have some other idea, we would love to hear them :)

**Comment by Shreya001** [comment]
Yes thats a good idea we can increase the support for the app,but i want to
know what will be the use of the list of trips we get?

**Comment by anubhakushwaha** [comment]
@Shreya001 It is for the convenience of the users, suppose I know I have two trips coming back to back, both of different durations and hence different requirements i.e no. of medicines and other stuff. By the current implementation, we can schedule only one trip and after we have been notified about it then only can we set a new trip reminder, was I clear this time? Let me know incase of more clarity :)

**Comment by Shreya001** [comment]
Yes i did,will do this by tommorrow, If any queties will ask you for details

**Comment by Shreya001** [comment]
I wanted to ask that everything is being added to the database so shld i
implement the class taking data from the database and the creating its
object and adding it to a list?

**Comment by Shreya001** [comment]
Hey,
I have a querey. What i did was made a new class trip where i intialized
the time, date, location and the items to be taken. I also overrided the
add function of an arraylist which adds the trip objects based on their
date and time. I have trouble figuring out that where do I use this array
to throw intents to use alarm.
Thanks

On 12-Sep-2017 11:30 PM, "Shreya Singal" <shreyashivam20@gmail.com> wrote:

I wanted to ask that everything is being added to the database so shld i
implement the class taking data from the database and the creating its
object and adding it to a list?

**Comment by anubhakushwaha** [comment]
@Shreya001 You can use https://stackoverflow.com/questions/12785702/android-set-multiple-alarms , just have a different request code each time, let me know if it helps and in any case we can have a discussion at slack :) Feel free to have a discussion

**Comment by Shreya001** [comment]
@anubhakushwaha i dont know the reminders appear but keep crashing.
